cpp/tests/experimental/spatial/point_distance_test.cu

```diff
@@ -37,12 +42,73 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include <numeric>
+
 namespace cuspatial {
 
+using namespace array_view;
+
+/**
+ * @brief Helper function to generate a random point
+ */
+template <typename T>
+struct RandomPointGenerator {
```

Comment:
Perhaps this should be in our random.cuh test utility file?

Reply:
I believe that's currently in the benchmark utility. I'd love to see it shared across between benchmark and tests though.

Reply:
It's header-only. Should I directly include that file here or make a copy of the header to `test/utility`?

Reply:
We should think about where things that are used for both test and benchmark go...

Reply:
In cudf that would be `cudftestutils`, export it as a library with cmake and build benchmark with it as a dependency. We could do the same?

Reply:
`include/testutils/random.cuh` -- no need to repeat "cuspatial" in the path, since you already have to include it with `#include <cuspatial/include/...>`

Reply:
Unfortunately CMakeLists only adds `cuspatial/cpp` into the include directories. At best you can do `include/testutils/random.cuh`. I couldn't find the variable name that points to the top level directory.

Reply:
Chatted with @vyasr - we should just follow how cudf include the testutil headers, under`cudf_test`. We can do `cuspatial_test`.